EPPs/convert_and_dispatch_genotypes.py

```diff
@@ -344,14 +389,15 @@ def main():
 
     p = Process(lims, id=step_id)
 
-    if args.genotypes_artifact_id:
-        a = Artifact(lims, id=args.genotypes_artifact_id)
-        input_genotypes_content = StringIO(lims.get_file_contents(uri=a.files[0].uri))
```

Comment:
It looks like we're now passing file paths directly, rather than an artifact ID - is this an intended usage change?